README.rst

```diff
@@ -0,0 +1,141 @@
+
+
+LICENSE
+------------------------------------------------------------------------
+
+This software is subject to license and copyright restrictions listed
+`here. <https://github.com/diffpy/diffpy.pdfmorph/blob/master/LICENSE.txt/>`_
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+The PDFmorph application has a Command Line interface. If you are 
+unfamiliar with the terminal or windows command prompt, it is recommended
+that you consult online resources and become somewhat familiar before
+using PDFmorph.
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - `shared helper utilities <https://github.com/diffpy/diffpy.utils/>`_ for wx GUI
+
+To install these packages, please see installation instructions at 
+their respective web-pages.
+
+
+INSTALLATION
+------------------------------------------------------------------------
+
+To create and activate a conda environment to use this software, run 
+the following command from the command line ::
+	
+	conda create -n pdfmorph_env python=3
+	source activate pdfmorph_env
+
+If you're using Windows, replace ``source activate pdfmorph`` with ::
+	
+	activate pdfmorph_env
+
+When you are finished with the session, exit the environment by running :: 
+
+	source deactivate pdfmorph_env
+
+or ::
+
+	deactivate pdfmorph_env
+
+If you use this conda environment, make sure to run the ``source activate 
+pdfmorph_env`` or ``activate pdfmorph_env`` commands from your command 
+line before every session using the applicaiton.
+
+If you're using Anaconda Python, you can install from the "conda-forge" 
+channel of Anaconda packages ::
+
+     conda config --add channels conda-forge
+     conda install diffpy.pdfmorph
+
+If you don't use Anaconda or prefer to install from sources, please 
+consult online documentation.
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+
+USING PDFmorph
+------------------------------------------------------------------------
+
+For detailed instructions and full tutorial, consult online documentation.
+
+Once the required software, including PDFmorph is all installed, open
+up a terminal and check installation has worked properly by running ::
+
+	source activate pdfmorph_env
+	pdfmorph -h			#get some helpful information
+	pdfmorph --version
+
+If installed correctly, this last command should return the latest 
+version of PDFmorph. To begin using PDFmorph, run a command like ::
+
+	pdfmorph <target PDF file> <morphed PDF file>
```

Comment:
the users will need to know what kind of file so add words like "where PDF file is a text file that contains a PDF, such as produced by ``PDFgetX2``, ``PDFgetX3``, or ``PDFgui``"

Reply:
done